app/views/content/funding-your-training.md

```diff
@@ -13,9 +13,9 @@
   jump_links:
     Tuition fee and maintenance loans: "#tuition-fee-and-maintenance-loans"
     Bursaries and scholarships: "#bursaries-and-scholarships"
-    Extra financial support for parents and carers: "#extra-financial-support-for-parents-and-carers"
-    Extra financial support for students with disabilities: "#extra-financial-support-for-students-with-disabilities"
-    Applying for funding if you come from outside England: "#applying-for-funding-if-you-come-from-outside-england"
+    Extra financial support for parents and carers: "#if-youre-a-parent-or-carer"
```

Comment:
@EmmaFrith are you happy with these headings on the right hand side? Given we've amended these below.

Reply:
oh crap.